test/test.js

```diff
@@ -37,12 +37,17 @@ function getTestOptions (settings) {
     return defaults({}, settings && settings.set, disabled);
 }
 
+function writeConfig (from, to) {
+    writeFile(`${to}/.git/config`, readFile(`${to}/.git/config`) + readFile(`${from}/gitconfig`));
```

Comment:
Strange, local `.gitconfig` works perfectly for me. Are you sure it is correctly written to the testing repo? Have you tried putting it to the testing repo manually and running `git branch`?